Formula/mmseqs2.rb

```diff
@@ -0,0 +1,64 @@
+class Mmseqs2 < Formula
+  desc "Software suite for very fast protein sequence search and clustering"
+  homepage "https://mmseqs.org"
+  url "https://github.com/soedinglab/MMseqs2/archive/1-c7a89.tar.gz"
+  version "1-c7a89"
+  sha256 "e756a0e5cb3aa8e1e5a5b834a58ae955d9594be1806f0f32800427c55f3a45d5"
+
+  head do
+    url "https://github.com/soedinglab/MMseqs2.git"
+    resource "documentation" do
+      url "https://github.com/soedinglab/MMseqs2.wiki.git"
+    end
+  end
+
+  bottle :disable, "Requires SSE4.1 support"
+
+  option "with-avx2", "Build with AVX2 instruction set support"
+
+  needs :openmp
+  depends_on :mpi => [:cxx, :optional]
+
+  depends_on "cmake" => :build
+  depends_on :arch => :x86_64
+
+  resource "documentation" do
+    url "https://github.com/soedinglab/MMseqs2.wiki.git", :revision => "6dbd3666edb64fc71173ee714014e88c1ebe2dfc"
+  end
+
+  def install
+    inreplace "src/version/Version.cpp", /.+/m, "const char *version = \"#{version}\";"
```

Comment:
Works for me. Please add a comment, and consider reporting upstream so that the tarball can be built directly.

Reply:
I added a comment and also improved the MMseqs2 cmake script in commit f7991db. With the next MMseqs release I will be able to pass a cmake flag instead of this hack.